### PR TITLE
Makefile: add missing GRPC protos dependency

### PR DESCRIPTION
**Pull request by benesch**
Generating GRPC protos depends on protoc-gen-grpc-gateway, which is
installed by BOOTSTRAP_TARGET. That prerequisite wasn't properly
specified.

Release note: None

### CODE REVIEW

**Comment by benesch** [comment]
Fixes https://teamcity.cockroachdb.com/viewLog.html?buildId=497528&buildTypeId=Cockroach_UnitTests_BuildTestBinary&tab=buildLog.

**Review by nvb — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -447,7 +447,7 @@ CMAKE_SYSTEM_NAME := Windows
 endif
 
 CONFIGURE_FLAGS += --host=$(XHOST_TRIPLE) CC=$(CC_PATH) CXX=$(CXX_PATH)
-CMAKE_FLAGS += -DCMAKE_C_COMPILER=$(CC_PATH) -DCMAKE_CXX_COMPILER=$(CXX_PATH) -DCMAKE_SYSTEM_NAME=$(CMAKE_SYSTEM_NAME)
+XCMAKE_FLAGS := $(CMAKE_FLAGS) -DCMAKE_C_COMPILER=$(CC_PATH) -DCMAKE_CXX_COMPILER=$(CXX_PATH) -DCMAKE_SYSTEM_NAME=$(CMAKE_SYSTEM_NAME)
```
Comment (nvanbenschoten) [inline Makefile]: Is `XCMAKE_FLAGS` standard? If not, consider adding a comment explaining that the variable contains cross-compilation-specific flags.

**Review by nvb — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (benesch) [reply Makefile]: Not even a little bit standard, but then again none of this stuff is. Good point, though: a comment is always appreciated. Done!

**Comment by mberhault** [comment]
LGTM. I'm guessing this is the reason for the nightly failures.

**Comment by benesch** [comment]
Yerp. Thought I had hit the button last night, but no such luck.

**Pull request merged**